CHANGELOG.md

```diff
@@ -149,6 +149,7 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
   (e.g. `seqan3::WritableAlphabet` -> `seqan3::writable_alphabet`)! This change was motivated by the decision of the
   ISO C++ committee to also use snake case everywhere
   ([\#1235](https://github.com/seqan/seqan3/pull/1235)).
+* We now support Doxygen up to version 1.8.17.
```

Comment:
This needs to go to line 37 (this is the 3.0.1 changelog). Also the Link to the PR is missing.